kafka/src/main/java/org/apache/calcite/adapter/kafka/KafkaTableOptions.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.kafka;
+
+import org.apache.kafka.clients.consumer.Consumer;
+
+import java.util.Map;
+
+/**
+ * Available options for {@link KafkaStreamTable}.
+ */
+public final class KafkaTableOptions {
+  private String bootstrapServers;
+  private String topicName;
+  private KafkaRowConverter rowConverter;
+  private Map consumerParams;
+  //added to inject MockConsumer for testing.
+  private Consumer consumer;
+
+  public String getBootstrapServers() {
+    return bootstrapServers;
+  }
+
+  public KafkaTableOptions setBootstrapServers(final String bootstrapServers) {
+    this.bootstrapServers = bootstrapServers;
+    return this;
+  }
+
+  public String getTopicName() {
+    return topicName;
+  }
+
+  public KafkaTableOptions setTopicName(final String topicName) {
+    this.topicName = topicName;
+    return this;
+  }
+
+  public KafkaRowConverter getRowConverter() {
+    return rowConverter;
+  }
+
+  public KafkaTableOptions setRowConverter(
+      final KafkaRowConverter rowConverter) {
+    this.rowConverter = rowConverter;
+    return this;
+  }
+
+  public Map getConsumerParams() {
```

Comment:
What are key/value types ? `Map<String, String>` ?

Reply:
yes, should

> @XuMingmin can you please address Julian's comments in JIRA about proper attribution (especially in documentation) :
> 
> > Make sure to respect Kafka's branding. Each page in the documentation where Kafka is mentioned, the first mention should call it "Apache Kafka". I'd change the name of this case/commit to "Adapter for Apache Kafka".

I forget some places, let me go though it again

Reply:
> What are key/value types ? `Map<String, String>` ?

Should be, let me specify it explicitly.